test/clustertest/tests/BadCommandTest.cpp

```diff
@@ -14,6 +14,10 @@ struct BadCommandTest : tpunit::TestFixture {
                 BedrockTester& leader = tester.getTester(0);
                 BedrockTester& follower = tester.getTester(1);
 
+                ASSERT_TRUE(tester.getTester(0).waitForState("LEADING"));
+                ASSERT_TRUE(tester.getTester(1).waitForState("FOLLOWING"));
+                ASSERT_TRUE(tester.getTester(2).waitForState("FOLLOWING"));
```

Comment:
What do these additions check for?

Reply:
That the cluster is in the correct state. Mainly, that both followers are following, and we aren't in a state where leader is leading with one follower and a third server still starting up.

BadCommandTest is somewhat flaky and I think it's because we can crash a server before the last server is actually up and online, and then it doesn't handle everything quite as expected, so hopefully this makes this test a bit more reliable.